kncaputo/underrated
# Feature/user functionality

## What's this PR do?
- Add `Star Rating` component
- Fetch user ratings and display them on `movieDetails`
- Allow user ratings to be updated when logged in

## Where should the reviewer start?
- `StarRating.js` and `MovieDetails.js`

## How should this be manually tested?
- Log in as a user with username `olivia@turing.io` and password `pword`
- Click on a movies details to see the loaded star ratings
- Adjust a star rating and refresh the page/re log back in to see the rating persist

## Any background context you want to provide?
- Will need to add conditional rendering for the rating so when a user is not logged in, they will be prompted to log in when trying to rate a movie
